app/controllers/users_controller.rb

```diff
@@ -166,6 +166,10 @@ def edit_user_params
     params.require(:user).permit(:email, :first_name, :last_name)
```

Comment:
Since we're now allowing form submission for netid users, we should restrict the permitted params here in the controller.